homeassistant/components/technove/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for TechnoVE."""
```

Comment:
Oh and can we rename this file to `entity.py`?

Reply:
Done.